### DISCUSSION TITLE
Doubt Regrading behind commit

### DISCUSSION THREAD

**Discussion by UG-SEP** [MakeContributions/DSA#189] [Q&A] [answered]
Hey @ming-tsai ,
I have a doubt that if a contributor created a pull request 1 hour ago and we merge a pull request 30 min ago so the main/ or the new  branch of the contributor will be 1 commit behind so we should accept the PR or not 
Please fulfill my doubt 
Thanks

**Comment by ming-tsai** [comment, marked as answer]
If we don't have any conflict we could accept the PR

**Reply by UG-SEP** [reply]
ok
